tests/test.py

```diff
@@ -1340,6 +1340,31 @@ def testReadAcrastructInAcrastruct(self):
         self.assertEqual(row['empty'], b'')
 
 
+class BaseBinaryPostgreSQLMixin(BaseTestCase):
```

Comment:
if it is mixin then it should not be inherited from BaseTestCase to allow child classes inherit only methods of this mixin. It is allow to correctly inherit several classes with separate logic. If it should inherit from some other base class then it is "SpecificBaseClass". But it's is okay if one mixin inherited from another mixin which extends his API.
Good example of this way structuring we can see in [django views](https://github.com/django/django/blob/master/django/views/generic/detail.py), where exists one base class and a lot of mixins, which you can inherit and create your own special base class with blackjack and other things ))

Reply:
:thinking: I've inherited it from `BaseTestCase` so that calling `super().setUp()` and `super().checkSkip()` were predictable.

Would it be more accurate to call this class `BaseBinaryPostgreSQLTestCase` or something?

Reply:
I think yes, will be okay. User who will inherit from this class will undertstand that it some base class and probably as other in a project do something in `setUp` and override it